profile_bluesky/startup/instrument/devices/aps_source.py

```diff
@@ -22,7 +22,6 @@
 class UndulatorEnergy(PVPositioner):
     """
     Undulator energy positioner.
-
     Main purpose this being a PVPositioner is to handle the undulator backlash.
```

Comment:
replace with this text (and resolve the WHICH_WAY):
`Always move the undulator to final position from the WHICH_WAY direction, by applying a backlash (hysteresis) correction as needed.`